game-app/game-core/src/test/java/games/strategy/triplea/delegate/VictoryTest.java

```diff
@@ -529,4 +535,50 @@ void testNoPuResourcesToPurchase() {
     assertNull(error);
     assertEquals(italianResources, italians.getResources().getResourcesCopy());
   }
+
+  @Test
+  void testHasResourceWithOnePlayer() throws GameParseException {
+    final List<GamePlayer> players1 = List.of(italians);
+    /* italian starts with 16 PUs, 50 Fuel and 20 Ore */
+    /* testing default value "PUs" for hasResource */
+    attachment.setHasResource("15");
+    assertTrue(attachment.checkHasResource(players1));
```

Comment:
Extract to method doing the calls for default, one resource and multiple resources) so it can be reused in method `testHasResourceWithTwoPlayers()`